src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -73,10 +73,21 @@ const useFagsakApi = (
             return;
         }
 
+        if (context.begrunnelse.valideringsstatus == Valideringsstatus.IKKE_VALIDERT) {
```

Comment:
We use strict equality so use `===` for compare functions.

Reply:
ok